tools/nl/embeddings/build_embeddings.py

```diff
@@ -11,11 +11,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Build the embeddings index by concatenating various inputs."""
+
+# TODO: This file should be renamed build_embeddings.py once it is ready to
+# replace the existing build_embeddings.py file.
+# TODO: Consider adding the model name to the embeddings file for downstream
+# validation.
 
```

Comment:
This comment should now become obsolete, right?

Reply:
Fixed it in follow up PR.